icebox/icebox.py

```diff
@@ -1378,6 +1408,8 @@ def parse_db(text, device="1k"):
         (0, 330, 143): ("padin_glb_netwk", "6"), # (0 0)  (330 143)  (330 143)  routing T_0_0.padin_6 <X> T_0_0.glb_netwk_6
         (0, 331, 143): ("padin_glb_netwk", "7"),
     },
+    "lm4k": {
```

Comment:
To find these out, create designs using the pad input (SB_GB_IO) for each global input, plus two designs using the internal oscillators if they can drive a global. Then run then through icebox_explain to get the global buffer number, and the ".extra_bit" data is bit position numbers